src/sections/info.tsx

```diff
@@ -54,21 +53,21 @@ function InfoPage() {
       <div id="info">
         <div id="version">Version: {version}</div>
         {!!info &&
-          Object.entries(info).map(([key, value]) => {
+          Object.entries(info).map(([key, value], index) => {
```

Comment:
Using `index` as`key` might not be good if the order of items may change, can this happen here?

Reply:
gotcha ;)